loki/tools/tests/test_tools.py

```diff
@@ -174,6 +174,16 @@ def test_joinable_string_list_long():
     assert str(obj) == ref
 
 
+def test_joinable_string_list_exceeding_indentation():
+    items = ['a', 'b', 'c']
+    for depth in range(100):
+        # This should not throw an assertion
+        obj = JoinableStringList(
+            items, sep=' ', width=132,
+            cont=f' &\n{"  " * depth}&')
```

Comment:
Not sure I understand this - aren't you just setting a large continuation indentation, without ever needing it because your `sep` is still just `' '`? For 3 single-char elements that should generate (`a b c`) and not need the continuation at all - or am I Misreading this (quite likely)?

Reply:
Oh, you're absolutely right. The test did trigger the original assertion, which simply looked at the length of the continuation string, but it did not assess the underlying formatting problem due to the lack of causing line length overflow. Top commit includes a fix.